.gitmodules

```diff
@@ -1,3 +1,6 @@
 [submodule "submodules/katex-fonts"]
 	path = submodules/katex-fonts
 	url = https://github.com/KaTeX/katex-fonts
+[submodule "submodules/unicode-fonts"]
+	path = submodules/unicode-fonts
+	url = https://github.com/Khan/KaTeX-test-fonts.git
```

Comment:
I can move this repo to the `KaTeX` organization.

Reply:
It probably makes it sense to have it at `KaTeX/test-fonts` or `KaTeX/unicode-test-fonts`, wdyt?

Reply:
I think either is fine.

Reply:
As its current name is `KaTeX-test-font` and in the future maybe we can add other ASCII fonts support, I move for the former.